app/models/web_hook/event.rb

```diff
@@ -44,12 +44,10 @@ def has_transactional_callbacks?
     end
 
     def enqueue
-      if connection.try!(:transaction_open?) && !connection.current_transaction.state.finalized?
```

Comment:
Then you probably can also remove this line:
https://github.com/3scale/porta/blob/9b34dc3475cd2879b53f92b439dd5bb78098cd0a/app/models/web_hook/event.rb#L138